raqm.c

```diff
@@ -285,8 +290,7 @@ itemize_by_script(int bidirun_count,
                  FriBidiRun *bidiruns,
                  Run *runs)
 {
-    int i, j;
-    int run_count = 0;
+    int i, j, run_count = 0;
```

Comment:
This change is not really needed in this commit, mixing unrelated changes in the same commit makes it harder to review the code.